api/src/main/java/org/apache/commons/rdf/api/BlankNode.java

```diff
@@ -41,60 +41,51 @@
  * node identifiers, if any, therefore also depend on the concrete RDF syntax or
  * implementation.
- * 
+ *
  * Implementations that handle blank node identifiers in concrete syntaxes need
  * to be careful not to create the same blank node from multiple occurrences of
  * the same blank node identifier except in situations where this is supported
  * by the syntax. </blockquote>
- * 
- * A BlankNode object created through the
- * {@link RDFTermFactory#createBlankNode()} method must be universally unique,
- * and SHOULD contain a {@link UUID} as part of its
- * {@link #internalIdentifier()}.
- * 
- * A BlankNode object created through the
- * {@link RDFTermFactory#createBlankNode(String)} method must be universally
- * unique, but also produce the same {@link #internalIdentifier()} as any
- * previous or future calls to that method on that factory with the same
- * parameters. In addition, it SHOULD contain a {@link UUID} as part of its
- * {@link #internalIdentifier()}, created using
- * {@link UUID#nameUUIDFromBytes(byte[])} using a constant salt for each
- * instance of {@link RDFTermFactory}, with the given identifier joined to that
- * salt in a consistent manner.
- * 
  *
+ * A BlankNode SHOULD contain a {@link UUID} string as part of its
+ * universally unique {@link #uniqueReference()}.
+ *
+ * @see RDFTermFactory#createBlankNode()
+ * @see RDFTermFactory#createBlankNode(String)
  * @see <a href="http://www.w3.org/TR/rdf11-concepts/#dfn-blank-node">RDF-1.1
  * Blank Node</a>
  */
 public interface BlankNode extends BlankNodeOrIRI {
 
     /**
-     * Return a <a href=
-     * "http://www.w3.org/TR/rdf11-concepts/#dfn-blank-node-identifier" >unique
-     * label</a> for the blank node. This label is generated by either
-     * {@link RDFTermFactory#createBlankNode()} or
-     * {@link RDFTermFactory#createBlankNode(String)} and is unique within the
-     * context of the instance of the factory. In particular, successive calls
-     * to the {@link RDFTermFactory#createBlankNode(String)} method on a single
-     * factory with the same parameters MUST return BlankNode objects with
-     * identical internalIdentifiers, but the identifiers SHOULD be mapped to
-     * unique values in the context of the factory instance.
-     *
-     * IMPORTANT: This is not a serialization/syntax label, and there are no
-     * guarantees that it is a valid identifier in any concrete syntax. For an
-     * N-Triples compatible identifier use {@link #ntriplesString()}. For all
-     * other syntaxes, the result of this method must be sanitized to produce a
-     * valid concrete identifier if one is needed.
+     * Return a reference for uniquely identifying the blank node.
+     * <p>
+     * The reference string MUST be universally unique, e.g. blank nodes created
+     * separately in different JVMs or from different {@link RDFTermFactory}
+     * instances MUST NOT have the same reference string.
```

Comment:
This is slightly inconsistent with RDFTermFactory.createBlankNode(String) that says "SHOULD NOT" where it says "MUST NOT" here.